pipenv/project.py

```diff
@@ -199,6 +199,10 @@ def sources(self):
         else:
             return [{u'url': u'https://pypi.python.org/simple', u'verify_ssl': True}]
 
+    @property
+    def source_urls(self):
```

Comment:
The duplication still exists. It also prevents a breakage of encapsulation.

By the `cli` module needing to understand the structure of the source, the encapsulation of the `project` module is broken, where `source` is a concept. I suppose that might a be a better explanation for the method's existence.